docs/overview/script-storage.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Office Scripts file storage and ownership'
+description: 'Information about how Office Scripts are stored in OneDrive and transferred between owners.'
+ms.date: 11/13/2020
+localization_priority: Normal
+---
+
+# Office Scripts file storage and ownership
+
+Office Scripts are stored as **.osts** files in your OneDrive. The permissions of a script inside a workbook are limited to the permissions of that script's owner. The script **.osts** files carry whatever shared access and permissions that user's OneDrive allows.
```

Comment:
What are we trying to say by "The permissions of a script inside a workbook are limited to the permissions of that script's owner?